package/setup.py

```diff
@@ -284,7 +295,11 @@ def extensions(config):
     # build optimized versions of MDAnalysis.
     arch = config.get('march', default=False)
     if arch:
-        extra_compile_args.append('-march={}'.format(arch))
+        if using_msvc():
+            # MSVC doesn't have a good equivalent for the march flag
+            pass
+        else:
+            extra_compile_args.append('-march={}'.format(arch))
```

Comment:
FWIW, I don't have any particular objection to the approach from a quick glance.. CI would fail without this shim I guess?

I don't know if long-term dependence on `distutils` is really desirable but that's another issue. Maybe a `try..` style approach would be more robust, but not sure if we want to vendor one from upstream or can easily use another dep to provide that

Reply:
For clarity march build is disabled by being commented out `setup.cfg` in regular build so CI is normally fine.